clang/include/clang/CConv/CheckedRegions.h

```diff
@@ -83,8 +83,8 @@ class CheckedRegionFinder : public clang::RecursiveASTVisitor<CheckedRegionFinde
     bool containsUncheckedPtr(clang::QualType Qt);
     bool containsUncheckedPtrAcc(clang::QualType Qt, std::set<std::string> &Seen);
     bool isUncheckedStruct(clang::QualType Qt, std::set<std::string> &Seen);
-    bool isWild(std::set<ConstraintVariable*>&);
-    bool isWild(std::set<FVConstraint*>*);
+    bool isWild(const std::set<ConstraintVariable*>&);
+    bool isWild(const std::set<FVConstraint*>*);
```

Comment:
Can you remind me what `const` means when in this position? Is it saying that what is pointed to will not be changed (i.e., the contents of the set will be left alone) ?

Reply:
Yes, it says that the method will not mutate the set.